.gitignore

```diff
@@ -15,3 +15,6 @@ audit.log
 # Avoid ignoring Maven wrapper jar file (.jar files are usually ignored)
 !/.mvn/wrapper/maven-wrapper.jar
 .distribution
+
+# Ignore mounts
+.m2
```

Comment:
```suggestion
.m2
target
```

Reply:
target is already in the file, first line.